Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class GameManager : MonoBehaviour
+{
+
+    public GameObject Player;
+    public List<GameObject> CheckPoints;
+
+    private Vector3 _playerRespawnPosition;
+    private Quaternion _playerRespawnRotation;
+    private int _checkPointCount;
```

Comment:
Any reason for storing a count? Don't have store count if we use the mentioned implementation.